src/GraphQL/GraphQL.csproj

```diff
@@ -30,14 +30,14 @@
 
   <ItemGroup>
     <PackageReference Include="GraphQL-Parser" Version="3.0.0" />
-    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <PackageReference Include="System.Reactive.Core" Version="3.1.1" />
     <PackageReference Include="System.Reactive.Linq" Version="3.1.1" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.1'">
     <PackageReference Include="System.Reflection.Extensions" Version="4.3.0" />
-    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.0.0" />
+    <PackageReference Include="System.Reflection.TypeExtensions" Version="4.4.0" />
```

Comment:
`4.4.0` lowest support is `netstandard1.3` so this needs to stay at `4.0.0`  The `netstandard.2.0` section targets `4.4.0`.

Reply:
done, test also not passing